st2common/bin/st2-run-pack-tests

```diff
@@ -205,9 +214,14 @@ if [ "${JUST_TESTS}" = false ]; then
     fi
 
     # Install global test dependencies
+    INSTALLED_PIP_PACKAGES=$(pip list)
```

Comment:
Just a heads up - invoking pip is slow so we should try to avoid invoking it multiple times and try to do everything in one go where it makes sense and it doesn't sacrifice readability (similar to `apt-get` command runs, it involves reading a local database, etc. which is slow).